Zend/tests/traits/enum_underscore_as_name.phpt

```diff
@@ -13,6 +13,6 @@ namespace {
 
 ?>
 --EXPECTF--
-Deprecated: Using "_" as a class name is deprecated since 8.4 in %s on line %d
+Deprecated: Using "_" as a trait name is deprecated since 8.4 in %s on line %d
```

Comment:
The file name of this file should also be fixed as this seems like one of my copy paste mistakes